docs/docs/glossary.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Glossary
+lang: en-US
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy beyondcorp zero-trust reverse-proxy ztn zta
+description: A quick reference of commonly used terms.
+---
+
+# Glossary
+
+Pomerium's documentation use a lot of terminology specific to the networking and security space. This glossary defines common terms readers may be unfamiliar with. If you come across an unfamiliar term not listed in this page, please let us know in our [Discuss support forum][support] and we'll add it.
+
+[[toc]]
+
+## General
+
+### Access Token
+This is a general term referring to a string issued by an identifying service like an [identity provider] that validates the holder to have a specific set of permissions. Most of the access tokens discussed in our docs are [JWTs].
+
+### Identity Provider
+An identity provider (**IdP**) is used to [authenticate] a user, i.e. confirm their identity. Pomerium uses external IdPs to better integrate into existing environments, and to achieve strong separation of services.
```

Comment:
I think it's worth mentioning that typically an identity provider supplies single-sign on to organizations. And maybe have a link to that.

> Single Sign-On (SSO) is the most frequently asked for requirement by enterprise organizations looking to adopt new SaaS applications. SSO enables authentication via an organization’s Identity Provider (IdP), such as Google Workspace or Okta, as opposed to users or IT admins managing hundreds, if not thousands, of usernames and passwords.

Reply:
Adding as its own section.